base/int.jl

```diff
@@ -506,6 +506,7 @@ if nameof(@__MODULE__) === :Base
 end
 
 rem(x::T, ::Type{T}) where {T<:Integer} = x
+rem(x::Signed, ::Type{Unsigned}) = x % unsigned(typeof(x)) 
```

Comment:
Should probably also have the opposite `x % Signed`.

Reply:
done and tests are also done